src/utils/ResultListUtils.ts

```diff
@@ -23,6 +23,23 @@ export class ResultListUtils {
     return resultList ? !!resultList.options.enableInfiniteScroll : false;
   }
 
+  public static scrollToTop(root: HTMLElement) {
+    const resultList = ResultListUtils.getActiveResultList(root);
+    if (!resultList) {
+      new Logger(this).warn('No active ResultList, scrolling to the top of the Window');
+      return window.scrollTo(0, 0);
+    }
+
+    const scrollContainer = resultList.options.infiniteScrollContainer;
```

Comment:
Could this variable end up as `undefined`? Maybe if `enableInfiniteScroll` is false? If so, we should check it is truthy before accessing the `scrollTo` property.

Reply:
negative, if you look at the option there is a default value. If all fails, it will be the window